### ISSUE TITLE
0.0.8.4 Crash on import decorations

### ISSUE THREAD

**Issue by Layne66** [TanukiSharp/MHArmory#12]
[2019-01-05_21-22-51_348.log](https://github.com/TanukiSharp/MHArmory/files/2730077/2019-01-05_21-22-51_348.log)

here's my error log when I try to import my decorations, not sure whats happening but when I go back to 0.0.8.3 it doesn't crash. Even a fresh download of 8.4 does this.

**Comment by TanukiSharp** [comment]
Thanks you very much for the log, I'm going to investigate and come back to you very soon.
Sorry for the troubles.

**Comment by TanukiSharp** [comment]
I could identify the problem, reproduce it and fix it. There were indeed a crash issue in the decoration import feature introduced in version 0.0.8.4 when I added localization.

I made a version [0.0.8.6](https://github.com/TanukiSharp/MHArmory/blob/master/ChangeLog.md#0086-20190107) available that fixes the issue. Again, sorry for the trouble, and thank you to let me know if this indeed fixes the issue for you too.

Thanks.
